src/Compilers/Server/VBCSCompilerTests/VBCSCompilerServerTests.cs

```diff
@@ -350,7 +366,7 @@ public async Task CompilationExceptionShouldShutdown()
                     hitCompilation = true;
                     throw new Exception("");
                 });
-                using var serverData = await ServerUtil.CreateServer(compilerServerHost: compilerServerHost).ConfigureAwait(false);
+                using var serverData = await ServerUtil.CreateServer(EmptyCompilerServerLogger.Instance, compilerServerHost: compilerServerHost).ConfigureAwait(false);
```

Comment:
Was it intentional to not pass `Logger` to this test?

Reply:
Nope that was an ommission. Fixed.